app/models/club.rb

```diff
@@ -20,10 +20,10 @@ def activities_for_heatmap(
     Activity.where(
       type_id: [Activity.type_ids[:notice], Activity.type_ids[:reply]],
       created_at: [start_date..end_date])
-      .select('created_at')
-      .joins(user: :club).merge(Club.where(id: id))
-      .map { |n| n.created_at.to_i }
-      .inject(Hash.new(0)) { |h, tm| h[tm] += 1; h }
-      .to_json
+            .select('created_at')
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.